python/pyspark/sql/pandas/serializers.py

```diff
@@ -20,6 +20,9 @@
 """
 
 from pyspark.serializers import Serializer, read_int, write_int, UTF8Deserializer
+from pyspark.sql.types import DataType, UserDefinedType, StructType, \
+    _has_udt
```

Comment:
mypy lint
```
python/pyspark/sql/pandas/serializers.py:23: error: Module 'pyspark.sql.types' has no attribute '_has_udt'
```
I think it is wrongly reported by mypy. I don't know how to make it happy.